$NetBSD: patch-ac,v 1.5 2015/12/29 23:34:56 dholland Exp $

Don't use nonstandard headers.
In Standard C labels are not allowed to end blocks, so insert breaks.
Remove illegal use of FILE.

--- stalin-ia32.c.orig	2000-06-15 23:01:19.000000000 +0000
+++ stalin-ia32.c
@@ -1,7 +1,6 @@
 #include <stddef.h>
 #include <assert.h>
 #include <time.h>
-#include <alloca.h>
 #include <gc.h>
 #include <string.h>
 #include <math.h>
@@ -271935,6 +271934,7 @@ struct w10500 f11659(struct p11657 *p116
   /* x71541 stalin.sc:2538:88226 */
   break;
   default:
+  break;
   /* x111269 stalin.sc:10922:382727 */}
  /* x102535 */
  t56675 = p11661;
@@ -313620,6 +313620,7 @@ void f9302(struct p9300 *p9302, struct w
   /* x260584 stalin.sc:9753:340477 */
   break;
   default:
+  break;
   /* x260628 stalin.sc:2446:84988 */}
  /* x76986 */
  t63818 = p9310;
@@ -313748,6 +313749,7 @@ void f9302(struct p9300 *p9302, struct w
   /* x260592 stalin.sc:9753:340477 */
   break;
   default:
+  break;
   /* x260636 stalin.sc:2446:84988 */}
  /* x77072 */
  t63796 = p9318;
@@ -313998,6 +314000,7 @@ void f9302(struct p9300 *p9302, struct w
   /* x106836 stalin.sc:9753:340477 */
   break;
   default:
+  break;
   /* x71246 stalin.sc:2446:84988 */}
  /* x77166 */
  t63758 = p9340;
@@ -314071,6 +314074,7 @@ void f9302(struct p9300 *p9302, struct w
   /* x260596 stalin.sc:9753:340477 */
   break;
   default:
+  break;
   /* x260640 stalin.sc:2446:84988 */}
  /* x77193 */
  t63747 = p9344;
@@ -314140,6 +314144,7 @@ void f9302(struct p9300 *p9302, struct w
   /* x260600 stalin.sc:9753:340477 */
   break;
   default:
+  break;
   /* x260644 stalin.sc:2446:84988 */}
  /* x77220 */
  t63736 = p9348;
@@ -314209,6 +314214,7 @@ void f9302(struct p9300 *p9302, struct w
   /* x260590 stalin.sc:9753:340477 */
   break;
   default:
+  break;
   /* x260634 stalin.sc:2446:84988 */}
  /* x77247 */
  t63725 = p9352;
@@ -400667,6 +400673,7 @@ struct w10500 f1383(struct w190593 a1501
    /* x71315 stalin.sc:2453:85342 */
    break;
    default:
+   break;
    /* x71881 stalin.sc:2614:90587 */}
   /* x102338 */
   t76376 = p11628;
@@ -585808,7 +585815,6 @@ int main(int argc, char **argv)
  assert(offsetof(struct{char dummy; FILE *probe;}, probe)==4);
  assert(offsetof(struct{char dummy; unsigned int probe;}, probe)==4);
  assert(offsetof(struct{char dummy; unsigned int probe;}, probe)==4);
- assert(offsetof(struct{char dummy; FILE probe;}, probe)==4);
  assert(offsetof(struct{char dummy; struct p10810 *probe;}, probe)==4);
  assert(offsetof(struct{char dummy; struct p15702 *probe;}, probe)==4);
  assert(offsetof(struct{char dummy; struct p15698 *probe;}, probe)==4);
